src/components/header.js

```diff
@@ -39,7 +41,7 @@ function Header({ slug }) {
   const [isOpen2, setIsOpen2] = useState(false);
 
   /* first dropdown items */
-  const navDropdownLinks1 = ["link-one", "link-two"];
+  const navDropdownLinks1 = links.navDropdown1;
```

Comment:
Thanks for moving the links to the constants file! 

Also, can you use an array and mapping to create the first and second dropdown items, instead of hard coding? A user might want 3 nav drop downs for example, and shouldn't have to edit the code, only the the translation.json file

Reply:
Links in the constants file are refactored better (use arrays instead of incrementing variable names). I used mapping to create the dropdown items, so now a user can use 3 nav drop downs and it will be stored accordingly. I will work on dynamically creating the `NavDropDownButton`s in another PR.

Reply:
Great!!